File: docs/components_page/components/card/image_overlay.R
library(dashBootstrapComponents)

card <- dbcCard(
  list(
    dbcCardImg(
      src = "/static/images/placeholder286x180.png",
      top = TRUE,
      style = list("opacity" = 0.3)
    ),
    dbcCardImgOverlay(
      dbcCardBody(
        list(
          h4("Card title", className = "card-title"),
          p(
            paste(
              "An example of using an image in the background of",
              "a card."
            ),
            className = "card-text"
          ),
          dbcButton("Go somewhere", color = "primary")
        )
      )
    )
  ),
  style = list("width" = "18rem")
)

File: docs/components_page/components/card/image_overlay.jl
using DashBootstrapComponents

card = dbc_card(
    [
        dbc_cardimg(
            src = "/static/images/placeholder286x180.png",
            top = true,
            style = Dict("opacity" => 0.3),
        ),
        dbc_cardimgoverlay(
            dbc_cardbody([
                html_h4("Card title", className = "card-title"),
                html_p(
                    "An example of using an image in the background of " * "a card.",
                    className = "card-text",
                ),
                dbc_button("Go somewhere", color = "primary"),
            ],),
        ),
    ],
    style = Dict("width" => "18rem"),
)
